PresentData/PresentMonTraceConsumer.hpp

```diff
@@ -320,10 +342,11 @@ struct PMTraceConsumer
 
     void CompletePresent(std::shared_ptr<PresentEvent> p, uint32_t recurseDepth=0);
     std::shared_ptr<PresentEvent> FindBySubmitSequence(uint32_t submitSequence);
-    decltype(mPresentByThreadId.begin()) FindOrCreatePresent(EVENT_HEADER const& hdr);
-    decltype(mPresentByThreadId.begin()) CreatePresent(std::shared_ptr<PresentEvent> present, decltype(mPresentsByProcess.begin()->second)& presentsByThisProcess);
+    std::shared_ptr<PresentEvent> FindOrCreatePresent(EVENT_HEADER const& hdr);

```

Comment:
The change in return value is not strictly related to the PR, just syntactic sugar that I added when working on an abandoned approach to restore Vulkan tracking. I decided to put it in the PR anyway but feel free to mention if there are concerns.